### PR TITLE
Import history from Visual Studio Code (first part)

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -1500,6 +1504,7 @@ anbd->and
 ancapsulate->encapsulate
 ancesetor->ancestor
 ancesetors->ancestors
+ancester->ancestor
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: Also plural:
ancesters->ancestors

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -2045,6 +2051,7 @@ argumetn->argument
 argumetns->arguments
 argumnet->argument
 argumnets->arguments
+argumnts->arguments
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: Also singular:
argumnt->argument

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -5773,6 +5789,7 @@ containn->contain
 containner->container
 containners->containers
 containns->contains
+containr->container
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: Also plural:
containrs->containers

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -5911,6 +5928,8 @@ contraints->constraints
 contraitns->constraints
 contravercial->controversial
 contraversy->controversy
+contrbution->contribution
+contribtes->contributes
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: Also:
contribte->contribute
contribted->contributed

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -6404,7 +6423,10 @@ coyp->copy
 coypright->copyright
 coyright->copyright
 cpation->caption
+cpontent->content
 cpoy->coy, copy,
+cption->option
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: cption->option, caption,

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -14916,6 +14970,7 @@ mudules->modules
 muext->mutex
 muhammadan->muslim
 mulithread->multithread
+mulitiplier->multiplier
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: Also plural:
mulitipliers->multipliers

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -16062,6 +16126,7 @@ origionally->originally
 orign->origin
 orignal->original
 orignally->originally
+orignated->originated
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: Also:
orignate->originate
orignates->originates

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -16711,6 +16779,7 @@ peripathetic->peripatetic
 peripherial->peripheral
 peripherials->peripherals
 perisist->persist
+perisisted->persisted
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: Also:
perisistent->persistent

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -20932,6 +21010,7 @@ skipp->skip, skipped,
 skippd->skipped
 skipt->skip, Skype, skipped,
 skyp->skip, Skype,
+slaches->slashes
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: Also:
slach->slash

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -20969,6 +21048,7 @@ smething->something
 smll->small, smell,
 smller->smaller
 smoe->some
+smoketests->smoketest
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: Please check. This looks like the plural of smoketest.

File: codespell_lib/data/dictionary.txt [hunk 11]
```diff
@@ -21890,6 +21971,7 @@ sugestion->suggestion
 sugestions->suggestions
 sugests->suggests
 suggestes->suggests
+suggestins->suggestions
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: Also singular:
suggestin->suggestion

File: codespell_lib/data/dictionary.txt [hunk 12]
```diff
@@ -24302,6 +24391,7 @@ viatnamese->vietnamese
 vicefersa->vice-versa
 videostreamming->videostreaming
 vie->via
+viees->views
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: Also singular:
viee->view

**Comment by bl-ue** [comment]
Whew, conflicts resolved.

File: codespell_lib/data/dictionary.txt [hunk 13]
```diff
@@ -17573,6 +17616,8 @@ liftime->lifetime
 ligh->light, lie, lye,
 ligher->lighter, liar, liger,
 lighers->lighters, liars, ligers,
+lightbulp->lightbulb
+lightsensor->light sensor
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:17620]:
```suggestion
```

File: codespell_lib/data/dictionary.txt [hunk 14]
```diff
@@ -10435,6 +10464,7 @@ divsion->division
 divsions->divisions
 divsiors->divisors
 dloating->floating
+dmicrosoft->microsoft
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:10467]:
```suggestion
```

**Review by bl-ue — APPROVED** [review]

**Pull request merged**